src/dashboard/components/pages/Integrations.js

```diff
@@ -39,9 +39,14 @@ const Integrations = () => {
 		setGoogleCaptchaAPISecretKey( getOption( 'themeisle_google_captcha_api_secret_key' ) );
 	}, [ getOption( 'themeisle_google_captcha_api_site_key' ), getOption( 'themeisle_google_captcha_api_secret_key' ) ]);
 
+	useEffect( () => {
+		setStripeAPI( getOption( 'themeisle_stripe_api_key' ) );
```

Comment:
Saving the key in WP Option is not vulnerable since some plugins can scan the database?

Reply:
If you've the access to the code then you can do anything. We can always hash it so it doesn't come out in a db dump. Anyway, doesn't matter much as it's currently for testing.

Reply:
Yea, it doesn't seem to be an issue, it seems. All plugins have it out in the open in the database. We will also ask users to use restricted keys to stop them from giving us unlimited access just to view products, pricing, subscriptions & write sessions. But overall, all plugin, including WooCommerce, just saves it in the database.